templates/webapi/test/overview.html.ep

```diff
@@ -128,33 +128,35 @@
                 <div class="row" id="filter-modules">
                     <div class="col-4">
                         <div class="form-group">
-                            <label for="filter-modules"><strong>Module</strong></label>
+                            <label for="filter-modules"><strong>Module name</strong></label>
+                            <%= help_popover('Help for the <em>Module name</em> filter' => '
+                            <p>Shows jobs that contain the specified modules</p>') %>
                             <input name="modules" type="text" id="modules"
                             placeholder="modules, comma separated, e.g. mod1,mod2"
                             class="form-control">
                         </div>
                     </div>
                     <div class="col-4">
                         <div class="form-group">
-                            <label for="filter-module-re"><strong>Test module</strong></label>
+                            <label for="filter-module-re"><strong>Module code</strong></label>
+                            <%= help_popover('Help for the <em>Module code</em> filter' => '
+                            <p>Shows jobs that contain the modules found by grep command over module source code</p>') %>
                             <input type="text" class="form-control" name="module_re" placeholder="regular expression" id="filter-module-re">
                         </div>
                     </div>
                     <div class="col-2">
                         <div class="form-group">
-                            <label for="modules_result"><strong>Module Result</strong></label>
+                            <label for="modules_result"><strong>Module result</strong></label>
+                            <%= help_popover('Help for the <em>Module result</em> filter' => '
+                            <p>Show job with selected result.</p>
+                            <p>If module result is not selected, shows all the specified modules regardless of the results.</p>') %>
```

Comment:
I'd use "Show**s** job**s**" for consistency with the rest. The further paragraph can likely be removed as it should be clear anyways that empty boxes mean the filter isn't effective.
```suggestion
                            <p>Shows jobs with selected result.</p>') %>
```